### ISSUE TITLE
"Rate limited by github"

### ISSUE THREAD

**Issue by rfay** [mitchellh/golicense#5]
Mostly when I get this to work I get a bunch of things like `⚠️  github.com/aws/aws-sdk-go            rate limited by GitHub, waiting 46m59.238793s`

Seems this may not be altogether practical with github the way it is.

**Comment by mitchellh** [comment]
Is this with a GitHub token set as well? I've been running `golicense` pretty regularly and haven't hit this issue. Unauthenticated and you hit it after almost one use. 😄 But authenticated you should get 5000 req/hr which is hefty for what golicense is doing (about 1 req per dependency in general, maybe 2 at times).

**Comment by rfay** [comment]
No, it wasn't, I'd missed [the doc on that](https://github.com/mitchellh/golicense#github-authentication)

I recommend that you change the error to add the link in about GITHUB_TOKEN.
